serviceerror/already_exists.go

```diff
@@ -13,13 +15,20 @@ type (
 	}
 )
 
-// NewAlreadyExist returns new AlreadyExists error.
-func NewAlreadyExist(message string) error {
```

Comment:
We can't make backwards-incompatible changes to this package IMO (even if users shouldn't really be using it)

Reply:
It is not used by SDK. Only by server. But I agree... I should leave old constructor as deprecated.

Reply:
We do not know everyone that uses this. This is one of the most included libraries by Temporal users. Many users have proxies, client mocks, etc that may take advantage of these things. We should not adjust this library as if it's only used by server, github.com/temporalio/temporal is a better place for code that is only used by server.